cfgov/templates/wagtailadmin/login.html

```diff
@@ -1,10 +1,9 @@
 {% overextends "wagtailadmin/login.html" %}
-{% load compress static i18n %}
-
+{% load static i18n %}
 
 {% block furniture %}
     <div class="content-wrapper">
-        {% if messages  or form.non_field_errors %}
+        {% if messages or form.non_field_errors %}
```

Comment:
This logic has changed slightly [in 1.13](https://github.com/wagtail/wagtail/blob/v1.13.1/wagtail/wagtailadmin/templates/wagtailadmin/login.html#L14), now using `form.errors` here and just below.

Reply:
I haven't implemented this change as we handle field errors differently from vanilla Wagtail.